lib/api/src/module.rs

```diff
@@ -263,9 +263,11 @@ impl Module {
         resolver: &dyn Resolver,
     ) -> Result<InstanceHandle, InstantiationError> {
         unsafe {
-            let instance_handle =
-                self.artifact
-                    .instantiate(self.store.tunables(), resolver, Box::new(()))?;
+            let instance_handle = self.artifact.instantiate(
+                self.store.tunables(),
+                resolver,
+                Box::new(()),
+            )?;
```

Comment:
Looks like unrelated formatting changes, we use 1 behind latest stable (though I think we may not have updated to 1.50 in this repo yet), would be good to run that version of rustfmt on the code